docs/source/dev/supported_vrplib_fields.rst

```diff
@@ -26,6 +26,8 @@ PyVRP supports the following specifications:
    ``CAPACITY``
       Homogeneous capacity of all vehicles in the instance.
       The number of vehicles is provided using the :term:`VEHICLES` specification.
+      To specify heterogeneous capacities, see :term:`CAPACITY_SECTION`.
+      When :term:`CAPACITY` and :term:`CAPACITY_SECTION` are both provided, the data from :term:`CAPACITY_SECTION` is used.
```

Comment:
This is because `vrplib` assumes that specifications precede the data sections. On the PyVRP side, we don't need to change `read.py`.